ReactAndroid/src/main/java/com/facebook/react/views/recyclerview/RecyclerViewBackedScrollView.java

```diff
@@ -294,7 +297,8 @@ public int getItemCount() {
 
     @Override
     public long getItemId(int position) {
-      return mViews.get(position).getId();
+      // this should return the view's actual ID to work properly with the view we're extending.
```

Comment:
I think I can delete this comment.  It should return a unique value for the element in the list, which `reactTagFor` will provide.  `getId` isn't guaranteed to always return a value now.